src/OrbitGl/ScopeTreeTest.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "ScopeTree.h"
+
+using orbit_client_protos::TimerInfo;
+
+namespace {
+
+const TimerInfo& CreateTimerInfo(uint64_t start, uint64_t end) {
+  static std::vector<std::unique_ptr<TimerInfo>> timer_buffer;
+  auto timer_info = std::make_unique<TimerInfo>();
+  timer_info->set_start(start);
+  timer_info->set_end(end);
+  timer_buffer.push_back(std::move(timer_info));
+  return *timer_buffer.back();
+}
+
+TEST(ScopeTree, TreeCreation) {
+  ScopeTree tree;
+  EXPECT_EQ(tree.Size(), 1);
```

Comment:
Although we are having an initial dummy node, I would expect the size and the Depth to not include it.

Reply:
Good point, I changed Depth() to Height() which is the number of edges of the longest path, which fits with what you expect.